src/visualization/streamlit-v1.py

```diff
@@ -33,6 +33,12 @@ def gen_random_data():
 
 @st.cache
 def gather_umap_data():
+    '''
```

Comment:
These line character lengths are tiny. Update your flake8 config so you don't have to deal with this anymore.